geomesa-spark/geomesa-spark-jts/src/main/scala/org/locationtech/geomesa/spark/jts/udf/GeometricConstructorFunctions.scala

```diff
@@ -13,13 +13,15 @@ import org.locationtech.geomesa.spark.jts.util.GeoHashUtils._
 import org.locationtech.geomesa.spark.jts.util.SQLFunctionHelper._
 import org.locationtech.geomesa.spark.jts.util.{GeometryUtils, WKBUtils, WKTUtils}
 import org.locationtech.jts.geom._
+import org.locationtech.jts.io.geojson.GeoJsonReader
 
 object GeometricConstructorFunctions {
 
   @transient
   private val geomFactory: GeometryFactory = new GeometryFactory()
 
   val ST_GeomFromGeoHash: (String, Int) => Geometry = nullableUDF((hash, prec) => decode(hash, prec))
+  val ST_GeomFromGeoJSON: String => Geometry = nullableUDF(text => {new GeoJsonReader().read(text)})
```

Comment:
Actually, that's kinda wrong.  The reader has a GeometryFactory.

Other than that, each call to read gets its own JSON Parser.  Maybe this can be made static somewhere?  Thoughts?